helix-front/client/app/dashboard/dashboard.component.ts

```diff
@@ -172,17 +168,18 @@ export class DashboardComponent implements OnInit, AfterViewInit, OnDestroy {
       .subscribe(
         result => {
           _.forEach(result, (resource) => {
-            const newId = this.visNetworkData.nodes.getLength() + 1;
+            const lastId = this.nodes.length;
+            const newId = this.nodes.length + 1;
             this.resourceToId[resource.name] = newId;
-            this.visNetworkData.nodes.add({
+            (this.visNetworkData.nodes as Node[])[this.visNetworkData.nodes.length] = {
```

Comment:
These changes are to update to ngx-vis major version 3, which shipped breaking changes to it's API. Details here: https://github.com/apache/helix/issues/2092#issuecomment-1125486970

Reply:
The approach I used was to refer to the diff in the ngx-vis example code at the time of their breaking changes.  The changes here mirror the changes that the library authors made to their own examples. https://github.com/apache/helix/issues/2092#issuecomment-1126406709